worlds/checksmate/docs/checksmate_en.md

```diff
@@ -0,0 +1,66 @@
+# ChecksMate Chess Randomizer Setup Guide
+
+![A chess piece with several blurry opposing pieces in the background](https://i.imgur.com/fqng206.png)
+
+## Required Software
+
+- Any ChecksMate client. Currently, a modified ChessV client is supported and can be accessed via
+  its [Github releases Page](https://github.com/chesslogic/chessv/releases) (latest version)
+- Archipelago from the [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases)
+    - (select `ChecksMate Client` during installation.)
+
+## Configuring your YAML file
+
+### What is a YAML file and why do I need one?
+
+See the guide on setting up a basic YAML at the Archipelago setup
+guide: [Basic Multiworld Setup Guide](/tutorial/Archipelago/setup/en)
+
+Some releases of the ChecksMate client include an example YAML file demonstrating some supported options.
+
+### Where do I get a YAML file?
+
+You can customize your settings by visiting the [ChecksMate Player Settings Page](/games/ChecksMate/player-settings)
+
+Some examples of certain outcomes are available in [this valid players file](checksmate-example.yaml), which can be used
+to generate a multiplayer multiworld (but should instead be used for your own inspiration).
+
+#### Difficulty and Material via Progression Balancing
+
+Increased progression balancing will place your pieces earlier, including pocket pieces and consuls. However, the AI
+makes weaker chess moves for each useful item you acquire, so the game can be much easier with useful and filler items.
+Since progression balancing tends to move useful items later, you may actually find the game easier at lower progression
+balancing.
```

Comment:
StarCraft II also has useful items which make progression easier.

Reply:
I realize now this doesn't actually mean every Useful item, no matter what it is, impacts the AI. But rather, the only Useful item that exists at all, specifically does that. I think in _most_ cases the progression items will be stronger than the ELO Lobotomies, so I wonder if this paragraph is better off just being removed. Or, alternatively, you could go the route of locking the Useful items so they aren't progression balanced

Reply:
Locking them sounds like a cool idea. I suppose that might cause issues in some extreme progression balancing situations (e.g. 99). I wonder if it would ever practically be an issue though.

Reply:
Even in those cases, it doesn't stop generation. It'd be really rare anyway and it's not like the items would be useless either

Reply:
I am not otherwise under the impression that observations about the unique impact of progression balancing on the game in documentation about setup are necessarily a PR submission blocker.

I have removed the text.